src/odemis/acq/test/move_test.py

```diff
@@ -310,12 +311,18 @@ def setUpClass(cls):
 
```

Comment:
If I understand correctly, the idea is that the test case runs by default with a METEOR TFS-1, but if you've already started a simulator of a different METEOR that still works? It's strange, because `start_backend()` should detect this situation and automatically stop the current back-end and start the METEOR TFS-1 version.

I'd suggest to refactor this test case to have 2 different versions: one for TFS-1 and one for Zeiss-1. I think you can (again) use sub-classes to do that elegantly. You'd have a parent class which does the TFS-1, but it would use a few class constants like `.MIC_CONFIG` and `.ROTATION_AXES`, which can be overridden by the child classes. One of the child class would be the Zeiss-1 . It can also override some methods, to check more things on the Zeiss.